ecosystem/sep-0038.md

```diff
@@ -0,0 +1,200 @@
+Discussion: https://github.com/stellar/stellar-protocol/issues/901
+Version 1.0.0
+```
+
+## Summary
+
+This protocol enables anchors to accept off-chain assets in exchange for different on-chain assets, and vice versa. Specifically, it enables anchors to provide quotes that can referenced within the context of existing Stellar Ecosystem Proposals. How the exchange of assets is facilitated is outside the scope of this document.
+
+## Motivation
+
+Anchoring an asset and issuing an asset are distinct functions that have different business and technical requirements. However, issuing an asset has often been a prerequisite for anchoring an asset. This protocol enables anchors to transfer value on and off the Stellar network for their clients regardless of whether a one-for-one reserve-backed Stellar asset exists for the off-chain asset held by the anchor's clients.
+
+Removing this requirement for anchors also provides downstream benefits to ecosystem participants generally. Enabling anchors to accept any Stellar asset will naturally decrease liquidity fragmentation on the decentralized exchange, leading to greater market depth and tigher spreads between trading pairs.
+
+## Specification
+
+### Authentication
+
+All endpoints require authentication in the form of a [SEP-10](sep-0010.md) JSON Web Token (JWT) in the `Authorization` header of the request. 
+
+```
+Authorization: Bearer <jwt>
+```
+
+### Content Type
+
+All endpoints accept in requests the following `Content-Type`:
+- `application/json`
+
+All endpoints respond with content type:
+- `application/json`
+
+### Errors
+
+If an error occurs when calling any endpoint, an appropriate HTTP status code
+will be returned along with an error response.
+
+#### Status Code
+
+Common HTTP status codes may be returned for a server. In particular the following are expected:
+
+Status Code | Name | Reason
+-----|------|------------
+`400` | Bad Request | The request is invalid in any way.
+`403` | Permission Denied | No `Authorization` header has been provided or the contents of the header are not accepted as valid.
+
+The response body must contain a human-readable description of the reason for error:
+
+```json
+{
+  "error": "The requested asset is not supported. See GET /prices for supported assets."
+}
+```
+
+### Asset Identification Format
+
+This protocol can be used to provide quotes for any class of asset in exchange for a Stellar asset. The following format is used to identify an asset in API requests and responses.
+
+```
+<scheme>:<identifer>
+```
+
+The currently accepted `scheme` values are:
+
+Name | Description
+-----|------------
+`stellar` | Used for Stellar assets.
+`iso4217` | Used for fiat currencies.
+
+`stellar` Identifier Format
+
+The [SEP-11](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0011.md#asset) asset format.
+
+`iso4217` Identifier Format
+
+The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) code for the fiat currency.
```

Comment:
❗ I think it is worth calling out that it is three character alphabetic codes. Technically ISO4217 defines both alphabetic and numeric codes, and we should be explicit here about which we want people to use. I don't think we should support both.
```suggestion
The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) three-character currency code for the fiat currency.
```